src/components/tree-item/tree-item.scss

```diff
@@ -239,7 +251,7 @@ slot[name="actions-end"]::slotted(*),
     transform: rotate(180deg);
   }
 
-  .item--expanded > .node-container > & {
+  .item--expanded .node-container > & {
```

Comment:
@benelan can you double check this one? I noticed it was left intact [here](https://github.com/Esri/calcite-components/pull/6320/files).

Reply:
I added a div with the `item--expanded` class between the `:host` and the `node-container` elements. So the `node-container` is now the direct child of `item--expanded` and the grandchild of `:host`. That's why I had to change all of the `:host > .node-container` selectors. 

It's fine with the change if it makes it easier to read/discern, the way it was before is technically more performant but not enough to be noticible. I'm just a weirdo who considers CSS selector optimization lool

Reply:
Cool. I removed the direct descendant selector because now there is an added div in-between `.nodeAndActionsContainer`.